samplesheets/tests/test_views_api_taskflow.py

```diff
@@ -144,6 +235,646 @@ def test_post_created(self):
         self.assertEqual(response.status_code, 400)
 
 
+class TestIrodsRequestCreateAPIView(TestIrodsRequestAPIViewBase):
+    """Tests for IrodsRequestCreateAPIView"""
+
+    def test_create(self):
+        """Test post() in IrodsRequestCreateAPIView"""
+        self.assertEqual(IrodsDataRequest.objects.count(), 0)
+        self._assert_alert_count(CREATE_ALERT, self.user, 0)
+        self._assert_alert_count(CREATE_ALERT, self.user_delegate, 0)
+        url = reverse(
```

Comment:
If the URL for the requests is always the same within a test class, we can make these tests a bit more concise by defining it as `self.url` in `setUp()`.

I've just recently started doing this myself, as some test modules are startting to grow rather big (as you probably have noticed :))